src/main/scala/org/deepdive/extraction/ExtractorRunner.scala

```diff
@@ -449,12 +449,20 @@ class ExtractorRunner(dataStore: JdbcDataStore, dbSettings: DbSettings) extends
     log.info(s"File dumped to ${actualDumpedPath}")
     val splitPrefix = s"${actualDumpedPath}-"
     val linesPerSplit = task.extractor.inputBatchSize
+    val maxParallel = task.extractor.parallelism
+    val splitNum = maxParallel * 10
```

Comment:
@netj Sorry for the late response. Here I was trying to reduce the effect of the data skew:  sometimes one thread might be stuck in a chunk of data and all other threads are waiting for the slowest one to finish. With smaller splits the data skew should have less impact.  So the idea here is to find a balance between UDF start overhead and the data skew.

I also believed that `xargs` has some load balancing to distribute the input chunks but I am not sure there.